Files/BaseLayout.cs

```diff
@@ -189,11 +192,44 @@ private void UnloadMenuFlyoutItemByName(string nameToUnload)
         {
             var menuItem = this.FindName(nameToUnload) as DependencyObject;
             if (menuItem != null) // Prevent crash if the MenuFlyoutItem is missing
-                (menuItem as MenuFlyoutItem).Visibility = Visibility.Collapsed;
+                (menuItem as MenuFlyoutItemBase).Visibility = Visibility.Collapsed;
         }
 
         public void RightClickContextMenu_Opening(object sender, object e)
         {
+            if (App.CurrentInstance.ViewModel.WorkingDirectory.StartsWith(App.AppSettings.RecycleBinPath))
+            {
+                (this.FindName("EmptyRecycleBin") as MenuFlyoutItemBase).Visibility = Visibility.Visible;
+                (this.FindName("OpenTerminal") as MenuFlyoutItemBase).IsEnabled = false;
+                UnloadMenuFlyoutItemByName("NewEmptySpace");
```

Comment:
You will probably need to use the localized strings to find the right items here.

Reply:
I'm finding elements based on the "x:Name" attribute. That does not change based on the locale.